tests/test_plots.py

```diff
@@ -14,9 +13,7 @@ def data(seed=31):
     mu = rng.normal(size=(4, 100))
     theta = rng.normal(size=(4, 100, 7))
 
-    return xr.Dataset(
-        {"mu": (["chain", "draw"], mu), "theta": (["chain", "draw", "hierarchy"], theta)},
-    )
+    return convert_to_datatree({"mu": mu, "theta": theta})
```

Comment:
Not opposed to keeping the "hierarchy" name for the dims. It's just that I got an error when using `convert_to_datatree`

Reply:
The tuple syntax `(["chain", "draw"], mu)` is an xarray shortcut to avoid using both `data` and `dims` as arguments. We don't have that shortcut for arviz so we'd have to use `dims` as keyword argument. I'd also recommend using `from_dict` (or explicit converters) over the `convert_to_datatree` function as general rule.